pylatex/base_classes/containers.py

```diff
@@ -187,6 +187,25 @@ def dumps(self):
 
         return string
 
+class Fragment(Container):
+    """A LaTeX fragment container class for fragmented document construction."""
```

Comment:
```suggestion
    """A LaTeX fragment container class for fragmented document construction."""
```

Please add something like:
> This only provides logical wrapping of the items. The final document will look the same as if all items would not have been part of a container.